src/Discord.Net.Core/Entities/Guilds/AutoModeration/AutoModRule.cs

```diff
@@ -0,0 +1,102 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Discord
+{
+    /// <summary>
+    ///     Provides properties used to create a <see cref="IAutoModRule"/>.
+    /// </summary>
+    public class AutoModRule
```

Comment:
What is the point of this, when `IAutoModRule` is inherited by socket and rest counterparts without referencing this type?